+++
date = "2017-06-12"
tags = ["zcash"]
title = "zcash-z_sendmany-8gb"
slug = "zcash"
+++

https://forum.z.cash/t/what-is-utxo-and-its-erros-mean/8460/11  

zcashでmining(pool)したものを匿名のz-addressに入れようとすると、エラーが出ます。

```bash
$ zcash-cli z_sendmany $TAADR $ZADDR
"message" : "general exception: std::bad_alloc"
```

8GBのメモリがないと駄目らしいです。それかpoolでminingしたものはz-addressに入れられないとかかな。まだまだわからないことばかりです。しばらく使ってみないとなんとも言えませんし、慣れませんね。
